packages/suite/package.json

```diff
@@ -70,6 +71,7 @@
         "ua-parser-js": "^1.0.2",
         "uuid": "^8.3.2",
         "web3-utils": "^1.6.1",
+        "yup": "^0.32.11",
```

Comment:
no need to have `yup` dependency here, move it to validation package and here use just that validation package as dependency

Reply:
done